ta/pkcs11/src/pkcs11_token.h

```diff
@@ -9,6 +9,20 @@
 #include <tee_api_types.h>
 #include <tee_internal_api.h>
 
+/* Hard coded description */
+#define PKCS11_SLOT_DESCRIPTION		"OP-TEE PKCS11 TA"
+#define PKCS11_SLOT_MANUFACTURER	"Linaro"
+#define PKCS11_SLOT_HW_VERSION		{ 0, 0 }
+#define PKCS11_SLOT_FW_VERSION		{ PKCS11_TA_VERSION_MAJOR, \
+					  PKCS11_TA_VERSION_MINOR }
+
+#define PADDED_STRING_COPY(_dst, _src)					\
```

Comment:
Why not use an inline function instead and get rid of the legitimate checkpatch warning?

Reply:
right. thanks.